ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -211,6 +211,13 @@ tests:
       COLLECTION_METHOD: core_bpf
     workflow: stackrox-automation-flavors-ibmcloudz-e2e
   timeout: 5h0m0s
+- as: nightly-ibmcloudz-ebpf-qa-e2e-tests
+  cron: 0 7 * * *
```

Comment:
I think we have periodic jobs in `stackrox-stackrox-nightlies.yaml` file, not in `stackrox-stackrox-master.yaml` but I'd defer to @gavin-stackrox to confirm.